sources/tools/Stride.Importer.3D/MeshConverter.cs

```diff
@@ -214,6 +215,9 @@ private unsafe Model ConvertAssimpScene(Scene* scene)
                 if (meshIndexToNodeIndex.ContainsKey(i))
                 {
                     var meshInfo = ProcessMesh(scene, scene->MMeshes[i], meshNames);
+                    
+                    if (meshInfo == null)
+                    {continue;}
```

Comment:
Style issue

Reply:
Would it be prefered to have 

`if (meshInfo == null) continue;` 

or

```csharp
if (meshInfo == null)
{
    continue;
}
```

Reply:
done!